### ISSUE TITLE
Cannot resolve module 'i18next-client'

### ISSUE THREAD

**Issue by cyrilf** [opening-hours/opening_hours.js#232]
Here is my error message:
`Cannot resolve module 'i18next-client' in /home/cyrilf/super-secret-project/app/node_modules/opening_hours/locales
 @ ./~/opening_hours/locales/core.js 154:19-44`

And I think it's related to the [peer dependencies not being installed](https://stackoverflow.com/a/35738361/1410020) as you can see:

```$ npm install opening_hours   
super-secret-project@2.3.7 /home/cyrilf/super-secret-project/app
├── UNMET PEER DEPENDENCY country-language@^0.1.7
├── UNMET PEER DEPENDENCY i18next-client@^1.11.1
├── opening_hours@3.5.0 
├── UNMET PEER DEPENDENCY react@15.4.2
└── UNMET PEER DEPENDENCY react-dom@15.4.2
```

`i18next-client` is [deprecated](https://www.npmjs.com/package/i18next-client) anyway, any plan on upgrading it to `i18next`?

Issue related to this one #187

**Comment by ypid** [comment]
The lib is tested on travis (ref: https://github.com/opening-hours/opening_hours.js/blob/master/.travis.yml) so it is possible to get it working.

**Comment by zavadpe** [comment]
@ypid `i18next-client` is not available on npm anymore. It has been deprecated (long time ago) and one should use `i18next` instead. Right now, library is unusable (at least in react project). Any plans to upgrade?

**Comment by ypid** [comment]
Thanks for the info. Still, "I would be happy over a PR".

**Comment by johhansantana** [comment]
getting this error as soon as I `import opening_hours from "opening_hours";` in a react native project

```
error: bundling failed: Error: Unable to resolve module `i18next-client`
```

any workaround?

**Comment by andidev** [comment]
@johhansantana 
`npm install i18next-client --save` worked for me as a solution (since it seems to be a peer dependency)

@zavadpe 
`i18next-client ` is deprecated but it is available in npm repo. I got it working on an React project by installing i18next-client myself. 

Though probably it would make sense to update this dependency since i18next-client is deprecated since at least 2 years.

**Comment by ypid** [comment]
Fixed by #385. Feel free to test it using https://www.npmjs.com/package/opening_hours/v/3.6.0-beta.3. Refer to #360 for testing generic feedback.
